### PR TITLE
Cleanup package to get ready for testing.

### PR DESCRIPTION
**Pull request by janwijbrand**

### CODE REVIEW

**Review by mgedmin — APPROVED** [review]
LGTM, except that the tests fail on Travis.

**Comment by mgedmin** [comment]
I haven't looked deeply, but this looks like fallout from zope.testbrowser 5.0.0.

https://icemac15.wordpress.com/2010/07/10/appswordpressicemac20100710get-rid-of-zope-app-testing-dependency/ might help fix it.

**Comment by janwijbrand** [comment]
The actual fixing is (should) now be done. When travis is green, the PR can be reviewed.

File: buildout.cfg [hunk 1]
```diff
@@ -1,16 +1,13 @@
 [buildout]
 extends =
-  http://download.zope.org/zopetoolkit/index/1.1/ztk-versions.cfg
-  http://download.zope.org/zopetoolkit/index/1.1/zopeapp-versions.cfg
+  https://raw.githubusercontent.com/zopefoundation/zopetoolkit/master/ztk-versions.cfg
 develop = .
 parts = test
 versions = versions
-allow-picked-versions = false
+#allow-picked-versions = false
```
Comment (thefunny42) [inline buildout.cfg]: I guess this needs to be cleaned.

**Review by thefunny42 — APPROVED** [review]
LGTM

File: buildout.cfg (hunk 1 above)
Comment (janwijbrand) [reply buildout.cfg]: remove those options.

**Pull request merged**